feature/google/src/test/java/BillingWrapperCommonTest.kt

```diff
@@ -168,77 +65,52 @@ class BillingWrapperTest {
     }
 
     @Test
-    fun defersCallingSkuQueryUntilConnected() {
-        mockStandardSkuDetailsResponse()
+    fun defersCallUntilConnected() {
         every { mockClient.isReady } returns false
 
-        val productIDs = setOf("product_a")
-
-        wrapper.querySkuDetailsAsync(
-            ProductType.SUBS,
-            productIDs,
-            {
-                this@BillingWrapperTest.storeProducts = it
-            }, {
-                fail("shouldn't be an error")
-            })
+        val token = "token"
+        var consumePurchaseCompleted = false
+        wrapper.consumePurchase(token) { _, _ ->
+            consumePurchaseCompleted = true
+        }
 
-        assertThat(storeProducts).`as`("SKUDetailsList is null").isNull()
+        assertThat(consumePurchaseCompleted).isFalse
 
         every { mockClient.isReady } returns true
 
         billingClientStateListener!!.onBillingSetupFinished(billingClientOKResult)
 
-        assertThat(storeProducts).`as`("SKUDetailsList is not null").isNotNull
+        assertThat(consumePurchaseCompleted).isTrue
     }
 
     @Test
     fun canDeferMultipleCalls() {
-        mockStandardSkuDetailsResponse()
         every { mockClient.isReady } returns false
 
-        val productIDs = setOf("product_a")
+        val token = "token"
 
-        wrapper.querySkuDetailsAsync(
```

Comment:
some existing tests were "general behavior of BillingWrapper" tests, i.e. about the connection logic. most of these used querySkuDetailsAsync as their example call into BillingWrapper, but which method was called didn't seem to matter. so to keep those tests in the main BillingWrapperTest, i used consumePurchaseAsync -- that method is exactly the same between BillingClient versions